deeplearning4j-core/src/main/java/org/deeplearning4j/nn/layers/BaseLayer.java

```diff
@@ -68,13 +69,17 @@ public BaseLayer(NeuralNetConfiguration conf, INDArray input) {
         this.conf = conf;
     }
 
+    protected LayerConfT layerConf() {
```

Comment:
Here's the new way for layer execution code to obtain its corresponding layer configuration in a typed manner.